src/components/Requirements/Requirements.vue

```diff
@@ -37,14 +36,25 @@
       </div>
       <div class="see-all-padding-x py-3">
         <h1 class="title">{{ showAllCourses.name }}</h1>
+        <div class="see-all-pages">
+          <span class="see-all-pageCount">{{ pageText }}</span>
+          <div class="see-all-buttonWrapper">
+            <button class="see-all-button" :class="{ 'see-all-button--disabled': !hasPrevPage }">
+              <span class="see-all-button-text" @click="prevPage()">PREV</span>
+            </button>
```

Comment:
Thank you for doing buttons instead of divs haha. Can you move `@click` to the `button` tag? Atm, I'm able to tab but I can't press `Enter` key to click the PREV or NEXT buttons.

Reply:
Shoot did not even realize I put them on the spans, thank you 😄